.lingma-agents.yaml

```
@@ -0,0 +1,44 @@
+    # 问题等级设置为 BLOCKER/CRITICAL/MAJOR/MINOR
+    problemLevel: BLOCKER
+    # 内容过滤配置
+
+  contentFilter:
+    fileGlobs:
+    - "**/pojo/**"
+    - "**/dto/**"
+    - "**/vo/**"
+
+  pathInstructions:
+    - pathSelectors:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)